### PR TITLE
initializing storybook

### PR DESCRIPTION
**Pull request by Hamza-A**
Initializing storybook

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials"
+  ]
+}
```
Comment (Fuzznimp) [inline .storybook/main.js:10]: End of the line

Comment (Hamza-A) [reply .storybook/main.js]: OK fixed 62f187d

File: .storybook/preview.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const withThemeProvider=(Story,context)=>{
+  return (
+    <ThemeProvider theme={lightTheme}>
+      <GlobalStyle />
+      <Story {...context} />
+    </ThemeProvider>
+  )
+}
+export const decorators = [withThemeProvider];
```
[the remark this answers is not in the document]
Comment (Hamza-A) [reply .storybook/preview.js]: OK fixed 0619bb3

File: package.json [hunk 3]
```diff
@@ -27,10 +29,15 @@
     "swr": "0.3.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.11.6",
```
[the remark this answers is not in the document]
Comment (Fuzznimp) [reply package.json]: I meant for all the dependencies :).

File: tsconfig.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noEmit": true,
-    "esModuleInterop": true,
+    "esModuleInterop": true,
```
[the remark this answers is not in the document]
Comment (Hamza-A) [reply tsconfig.json]: OK fixed 8cfd928

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (Hamza-A) [reply package.json]: OK fixed 4e8e6d4

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**